colorizer_data/types.py

```diff
@@ -120,16 +120,22 @@ class FrameDimensions(TypedDict):
     """Height of a frame in physical units (not pixels)."""
 
 
-class BaseMetadataJson(TypedDict):
-    """JSON dictionary format for `BaseMetadata`."""
+@dataclass
+class Frames3dMetadata(DataClassJsonMixin):
+    dataclass_json_config = config(letter_case=LetterCase.CAMEL, undefined=None)[
```

Comment:
nit: This line made me pause a second to read. Would it be clearer to break down this operation into two steps? like:
```
config = config(letter_case=LetterCase.CAMEL, undefined=None)
dataclass_json_config = config_object["dataclasses_json"]

```